src/backend/marsha/core/serializers/video.py

```diff
@@ -764,3 +765,98 @@ def get_lti_url(self, obj):
             self.context["request"],
             reverse("video_lti_view", args=[obj.id]),
         )
+
+
+class SharedLiveMediaSerializer(serializers.ModelSerializer):
+    """Serializer to display a shared live media model."""
+
+    class Meta:  # noqa
+        model = SharedLiveMedia
+        fields = (
+            "active_stamp",
+            "id",
+            "is_ready_to_show",
+            "pages",
+            "show_download",
+            "title",
+            "upload_state",
+            "urls",
```

Comment:
I'm wondering if `pages_urls` or simply `pages` wouldn't be more clear ?

Reply:
`pages` definitely not with the discussion we had about storing the number of pages in a document.
`urls` is the field name used in all other serialized objects.

But I'm open to the discussion if you really thing the name is not accurate

Reply:
+1 on just urls as on other objects

Reply:
Sorry, but I don't agree with you:
For a video `urls` contains different resolutions of the same instance. They represent the same content.
Here, `urls` contains different objects representations, as every page has different content.
The purpose of urls here, is not the same as a video, it shouldn't be named the same way, IMHO.

Reply:
Ah 🤔 another way to see it is: these are the urls to render the object? Should we nest "pages" in "urls"?
urls : {pages: {... }}

Reply:
Let's say we made a request to get a SharedLiveMedia object.
Now, we have `sharedLiveMedia.urls` which seems confusing to me.
@sampaccoud  you propose `sharedLiveMedia.urls.pages` which seems a bit strange, and no other key would exists under urls.
I still think that `sharedLiveMedia.pages_urls` or  `sharedLiveMedia.pages` would be more intuitive.

Sorry for nitpicking @lunika , but I believe that the api we provide deserves all our attention.

Reply:
Need to discuss irl as I don't get why it can be confusing

Reply:
So ? What should I use ? I forget that the original document can be downloaded, so something like this ?

```
urls: {
  pages: {},
  media?: string
}
```

Reply:
+1

Reply:
Done